distribution/docs/src/main/resources/_contents/_configuring/configuring-product-cache-contents.adoc

```diff
@@ -0,0 +1,20 @@
+
+==== Configuring the Product Cache
+
+All caching properties are part of the <<ddf.catalog.resource.download.ReliableResourceDownloadManager,Resource Download Settings>>.
```

Comment:
This is configured using the admin console so a sub-section of 3.1. might be a better location for it.

Also, the link is broken. I'm not sure how useful this sentence is anyways. We may want to explain what this does, where to find it and why it would need to be changed instead.